server/core/config/raw/pr_revision.go

```diff
@@ -0,0 +1,31 @@
+package raw
+
+import (
+	"os"
+
+	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/go-ozzo/ozzo-validation/is"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+)
+
+type PRRevision struct {
+	Username    string `yaml:"username" json:"username"`
+	PasswordEnv string `yaml:"password_env" json:"password_env"`
+	URL         string `yaml:"url" json:"url"`
+}
+
+func (p *PRRevision) Validate() error {
+	return validation.ValidateStruct(p,
+		validation.Field(&p.Username, validation.Required),
+		validation.Field(&p.PasswordEnv, validation.Required),
+		validation.Field(&p.URL, validation.Required, is.URL),
+	)
+}
+
+func (p *PRRevision) ToValid() valid.PRRevision {
+	return valid.PRRevision{
+		Username: p.Username,
+		Password: os.Getenv(p.PasswordEnv),
```

Comment:
Resolving the env variable here so that the temporal worker does not have to worry about resolving env variables. Open to other suggestions

Reply:
What's the issue with resolving it in the worker? It should just get picked up in the repo config.  We have other env vars in there already, this goes against the standard.

Reply:
Gotcha, I've changed it to resolve the env var in the repo config itself!